web/src/i18n/locales/en.json

```
@@ -3673,6 +3673,249 @@
+    "扩展价格": "Additional Pricing",
+    "额外价格项": "Additional price items",
+    "补全价格": "Completion Price",
+    "缓存读取价格": "Input Cache Read Price",
+    "缓存创建价格": "Input Cache Creation Price",
+    "缓存创建价格-5分钟": "Cache Creation Price (5-min)",
+    "缓存创建价格-1小时": "Cache Creation Price (1-hour)",
+    "缓存创建价格（5分钟）": "Cache Creation Price (5-min)",
+    "缓存创建价格（1小时）": "Cache Creation Price (1-hour)",
+    "分时缓存 (Claude)": "Timed Cache (Claude)",
+    "通用缓存": "Generic Cache",
+    "缓存读取": "Cache Read",
```

Biome (2.4.11)

[error] The key 缓存读取 was already declared.

(lint/suspicious/noDuplicateObjectKeys)